InterfaceCompatibleSignalBus.cs

```diff
@@ -25,7 +24,14 @@ public static void Fire<T>(T t)
 	{
 		if (actions.TryGetValue(typeof(T), out var action))
 			action(t);
-		else Debug.LogError($"Type of {typeof(T)} is trying to be fired as signal but is not subscribed anywhere");
+		else
+		{
+#if UNITY_STANDALONE
+			UnityEngine.Debug.LogError($"Type of {typeof(T)} is trying to be fired as signal but is not subscribed anywhere");
```

Comment:
in case that the logs will be decoupled for unity, there should be made a specific log system that uses the preprocesor just a single time, so there is no need to add the preprocessor every time, the motivation is nice, but it would be cool having a "CutomDebug" static class for this method, in tha cases uer who extend this can use that class instead of debug log, this would also make the code more readable.

everything else in this script is ok!